equinox/nn/_spectral_norm.py

```diff
@@ -1,18 +1,19 @@
-from typing import Generic, Optional, TypeVar
+from typing import Callable, Generic, Optional, TypeVar
```

Comment:
`from collections.abc import Callable`. This one is deprecated.

Reply:
mypy leads me astray once again ("note: Did you forget to import it from "typing"? (Suggestion: "from typing import Callable")")